microsoft-edge/webview2/how-to/webdriver.md

```diff
@@ -229,6 +229,11 @@ For more info about the `DebuggerAddress` property of the `EdgeOptions` object,
 
 Congratulations!  You've successfully automated a WebView2 project and driven WebView2 by using Selenium and Microsoft Edge WebDriver, by attaching Microsoft Edge WebDriver to a running WebView2 app.
 
+### Note for using UWP projects and debugging
+Because of the way that UWP projects are launched the WebDriver is not able to automatically configure the debug settings on launch. You will need to use the registry and enable debugging for all WebView2 process on startup.
```

Comment:
```suggestion
Because of the way that UWP projects are launched, the WebDriver2 control is not able to automatically configure the debug settings on launch. You will need to use the registry and enable debugging for all WebView2 processes on startup.
```